spring-xsuaa/src/main/java/com/sap/cloud/security/xsuaa/token/authentication/XsuaaJwtDecoder.java

```diff
@@ -111,7 +113,7 @@ private Jwt verifyToken(JWT jwt) {
 			String jku = tokenInfoExtractor.getJku(jwt);
 			String kid = tokenInfoExtractor.getKid(jwt);
 			String uaaDomain = tokenInfoExtractor.getUaaDomain(jwt);
-			return verifyToken(jwt.getParsedString(), jku, kid, uaaDomain);
+			return verifyToken(jwt.getParsedString(), jku, kid, uaaDomain, getZid(jwt));
```

Comment:
remove jku variable from line 113 and adjust verifyToken method signature to not take jku parameter anymore since it is a private method?